Common Distributions-Distributions.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>I also don't think we need this one, it's very rarely used and I don't think we use it in the course. Less is more.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>Removed, agreed less is more</p>